cmd/kind/create/cluster/createcluster.go

```diff
@@ -50,6 +51,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to a kind config file")
 	cmd.Flags().StringVar(&flags.ImageName, "image", "", "node docker image to use for booting the cluster")
 	cmd.Flags().BoolVar(&flags.Retain, "retain", false, "retain nodes for debugging when cluster creation fails")
+	cmd.Flags().BoolVar(&flags.Wait, "wait", false, "Wait for master node to be ready")
```

Comment:
"wait for the control-plane nodes to be ready", since while we only ever have one node today, this will still be true and we will continue to use this flag for multi-node

Reply:
let's make this a time.Duration string as well, and then the default can be "0" for don't wait, that way the user can decide how long they want to wait before resuming whatever the rest of their test rig wants to do